src/components/sequencing/Sequences.svelte

```diff
@@ -22,13 +23,34 @@
 
   export let user: User | null;
 
+  let baseColumnDefs: DataGridColumnDef[];
+
   type CellRendererParams = {
     deleteSequence: (sequence: UserSequence) => void;
     editSequence: (sequence: UserSequence) => void;
   };
   type SequencesCellRendererParams = ICellRendererParams<UserSequence> & CellRendererParams;
 
-  const baseColumnDefs: DataGridColumnDef[] = [
+  /**
+   * Sort the sequence table with the current users sequences at the top.
+   * @param valueA
+   * @param valueB
+   */
+  function usernameComparator(valueA: string, valueB: string): number {
+    if (valueA === null && valueB === null) {
+      return 0;
+    }
+    if (valueA === null) {
+      return -1;
+    }
+    if (valueB === null) {
+      return 1;
+    }
+
+    return valueA === user?.id ? 1 : -1;
+  }
+
+  $: baseColumnDefs = [
```

Comment:
Sorry do you mind keeping the reactive statements grouped up? The `let`s are in between.